ReactWindows/ReactNative.Bridge/RemoteDebuggerJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace ReactNative.Bridge
+{
+    public class RemoteDebuggerJavaScriptBundleLoader : IJavaScriptBundleLoader
```

Comment:
I'm of a few minds about this. 1) in this case, the class would be duplicated across platform-specific projects because there's no platform-specific code. 2) this particular class is pretty trivial, so the duplication maybe isn't so bad. 3) using a Shared Project for consistency of where some platform specific impls live while still sharing code until they diverse may be a reasonable compromise. In this particular case, I'm happy to move this impl back to the UWP project and have a duplicate of this class be the first addition to a new "Classic Desktop" project where Win7 specifics would live. thoughts?